bin/log-performance-results.js

```diff
@@ -10,22 +10,25 @@ const [ token, branch, hash, baseHash, timestamp ] = process.argv.slice( 2 );
 
 const resultsFiles = [
 	{
-		file: 'post-editor-performance-results.json',
+		file: 'post-editor.results.json',
 		metricsPrefix: '',
 	},
 	{
-		file: 'front-end-block-theme-performance-results.json',
+		file: 'front-end-block-theme.results.json',
 		metricsPrefix: 'block-theme-',
 	},
 	{
-		file: 'front-end-classic-theme-performance-results.json',
+		file: 'front-end-classic-theme.results.json',
```

Comment:
is there a need to remove this? it was my assumption that we might end up with more test results in the future and this could help make it obvious which result belongs to which test.

it's fine to change it, and we can handle it later of course, but it has been helpful to me at least seeing all the information in the filename

Reply:
The idea here was that if CI zips it into `performance-results` folder, then it doesn't need a `.performance-results.js` suffix. On second thought, though, there's no zipping when running locally, and everything lands in the `artifacts` folder, so it would be better to keep that suffix AND add it to the raw result `json`s as well. Will make an update.

Reply:
https://github.com/WordPress/gutenberg/pull/48684/commits/7b1b36ce7d0dc295ca10534b1c87f53baef3c7fa